app/forms/openid_connect_authorize_form.rb

```diff
@@ -28,7 +28,7 @@ class OpenidConnectAuthorizeForm
   validates :nonce, presence: true, length: { minimum: RANDOM_VALUE_MINIMUM_LENGTH }
 
   validates :response_type, inclusion: { in: %w[code] }
-  validates :prompt, presence: true, inclusion: { in: %w[select_account] }
+  validates :prompt, presence: true, inclusion: { in: %w[login select_account], allow_nil: true }
   validates :code_challenge_method, inclusion: { in: %w[S256] }, if: :code_challenge
```

Comment:
Did you mean to add `allow_nil: true` here? I don't think this is doing anything. The option should go outside of the curly brackets, like this:
```ruby
validates :prompt, presence: true, inclusion: { in: %w[login select_account] }, allow_nil: true
```
But I don't think we want this option because that tells the app to skip the validation if the value is nil.

http://edgeguides.rubyonrails.org/active_record_validations.html#allow-nil

Reply:
this is no longer valid either.  I'm going to be defaulting it to select_account when it's nil